cmd/verify.go

```diff
@@ -87,45 +86,40 @@ func runVerify(vo options.VerifyOptions, args []string) error {
 		return fmt.Errorf("could not unmarshal policy envelope: %w", err)
 	}
 
-	diskEnvs, err := loadEnvelopesFromDisk(vo.AttestationFilePaths)
+	artifactDigestSet, err := cryptoutil.CalculateDigestSetFromFile(vo.ArtifactFilePath, []crypto.Hash{crypto.SHA256})
 	if err != nil {
-		return fmt.Errorf("failed to load attestation files: %w", err)
+		return fmt.Errorf("failed to calculate artifact digest: %w", err)
 	}
 
-	verifiedEvidence := []witness.CollectionEnvelope{}
-
-	if vo.RekorServer != "" {
-
-		artifactDigestSet, err := cryptoutil.CalculateDigestSetFromFile(vo.ArtifactFilePath, []crypto.Hash{crypto.SHA256})
-		if err != nil {
-			return fmt.Errorf("failed to calculate artifact file's hash: %w", err)
-		}
-
-		rc, err := rekor.New(vo.RekorServer)
-		if err != nil {
-			return fmt.Errorf("failed to get initialize Rekor client: %w", err)
-		}
-
-		digestSets := []cryptoutil.DigestSet{}
-		digestSets = append(digestSets, artifactDigestSet)
-
-		verifiers := []cryptoutil.Verifier{}
-		verifiers = append(verifiers, verifier)
+	subjects := []cryptoutil.DigestSet{artifactDigestSet}
+	for _, subDigest := range vo.AdditionalSubjects {
+		subjects = append(subjects, cryptoutil.DigestSet{crypto.SHA256: subDigest})
```

Comment:
This is less than ideal.

We probably need to be parsing `AdditionalSubjects` and making sure they're valid digests prior to using them